com.github.Inochi2D.Inochi-Creator.yml

```diff
@@ -0,0 +1,167 @@
+id: com.github.Inochi2D.Inochi-Creator
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ldc
+  - org.freedesktop.Sdk.Extension.dmd
+command: "inochi-creator"
+finish-args:
+    - "--device=all" #OpenGL rendering, webcams (may be required? not sure)
+    - "--socket=x11"
+    - "--share=ipc" # The X and Wayland sockets are permanently exposed, because kdialog takes longer to start if it only has X access, and doesn't work at all if only using Wayland.
+    - "--socket=wayland"
+    - "--filesystem=host" #This is a dev tool, it should probably have as much access to the host as it can. It also doesn't yet utilize portals.
+    - "--env=XDG_CURRENT_DESKTOP=KDE" #Hopefully fixes the issues it has with running on non-KDE desktops.
+modules:
+  # --- Tools ---
+  - name: xprop
+    sources: 
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/app/xprop-1.2.4.tar.bz2
+        sha256: "8c77fb096e46c60032b7e2bde9492c3ffcc18734f50b395085a5f10bfd3cf753"
+
+  - name: zenity
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/zenity/3.41/zenity-3.41.0.tar.xz
+        sha256: 19b676c3510e22badfcc3204062d432ba537402f5e0ae26128c0d90c954037e1
+
+  - name: kdialog
+    buildsystem: cmake

```

Comment:
```suggestion
    buildsystem: cmake-ninja
```

Reply:
Wouldn't this build just fine with just `cmake`? Or is there another reason for using `cmake-ninja`? I'm not very sure on the differences between the two.

Reply:
it's faster. It's jusr cmake with the Ninja backend. It's prefered.